### ISSUE TITLE
nginx_config_template in Passengerfile.json is not using relative path

### ISSUE THREAD

**Issue by kriansa** [phusion/passenger#1557]
If you start passenger not by app root (where Passengerfile.json is located) then passenger standalone won't be able to find `nginx_config_template`, so you'll have to set manually by using `--nginx-config-template` in your command-line.

**Comment by FooBarWidget** [comment]
We'll have a look at this. For now, you can work around this issue with absolute paths.

**Comment by FooBarWidget** [comment]
This has been fixed in commit 900f3e6e.

**Comment by OnixGH** [comment]
This commit breaks the Node.js loader's loadApplication(), because startupFile is now absolute, appRoot was already absolute, so you get require("/home/user/app/"+"/home/user/app/x.js").
```javascript
var startupFile = PhusionPassenger.options.startup_file || 'app.js';
require(appRoot + '/' + startupFile);
```
